html/browsers/browsing-the-web/overlapping-navigations-and-traversals/cross-document-traversal-same-document-nav.html

```diff
@@ -38,9 +34,9 @@
   assert_equals(iframe.contentWindow.location.search, "?2");
   assert_equals(iframe.contentWindow.location.hash, "#3");
 
-  // Eventually ends up on ?2
-  await t.step_wait(() => iframe.contentWindow.location.search === "?2" && iframe.contentWindow.location.hash === "");
-}, "same-document traversals are not canceled by fragment navigations");
+  // Eventually ends up on ?1
+  await t.step_wait(() => iframe.contentWindow.location.search === "?1" && iframe.contentWindow.location.hash === "");
+}, "same-document traversals + fragment navigations");
```

Comment:
This is mostly just updating the actual expectations to match the test description text right? Before, it looks like we did expect same-document traversals to _not_ be canceled by fragment navigations, yet the actual expectation condition didn't match those words.

It looks like you fixed the condition, but why do we remove "are not canceled by" from the description?

Reply:
Before we were expecting them both to go through, in order: ?2 -> ?2#3 -> ?2. Now we are expecting them both to go through, but with the destination of the "back" calculated differently: ?2 -> ?2#3 -> ?1. So in neither case were we expecting a cancel, but the result is not really captured by "cancel vs. don't cancel", so I just made the test name vaguer, deferring all the explanation to the Jake digram.